splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -2145,4 +2146,349 @@ public static void INVALIDATE_GLOBAL_DICTIONARY_CACHE() throws Exception {
             }
         }
     }
+
+    public static void SHOW_CREATE_TABLE(String schemaName, String tableName, ResultSet[] resultSet) throws SQLException
+    {
+        try {
+            Connection connection = getDefaultConn();
+            Statement stmt = connection.createStatement();
+
+            EngineUtils.verifyTableExists(connection, schemaName, tableName);
+
+            ResultSet tableIdRs = stmt.executeQuery("SELECT T.TABLEID, T.TABLETYPE, T.COMPRESSION, T.DELIMITED, " +
+                    "T.ESCAPED, T.LINES, T.STORED, T.LOCATION" +
+                    " FROM SYS.SYSTABLES T, SYS.SYSSCHEMAS S " +
+                    "WHERE T.TABLETYPE IN ('T','E') AND T.SCHEMAID = S.SCHEMAID " +
+                    "AND T.TABLENAME = '" + tableName + "' AND S.SCHEMANAME = '" + schemaName + "'");
```

Comment:
the code change in ij.jj suggests that we could have commands like "show create table schema_name.*;", but this query does not seem to handle that.

Reply:
show create table schema_name.* is intending support. 
Clean these semi-finished codes and implement in a separate ticket.